style.css

```diff
@@ -0,0 +1,200 @@
+@import url('https://fonts.googleapis.com/css?family=Nunito:400,400i,700');
+/* Eliminate bullet point, spread navigation on a row */
+
+body {
+    font-family: 'Nunito', sans-serif;
+}
+.nav {
+    display: grid;
+    grid-template-columns: 60% 30% 10%;
+    list-style-type: none;
+    font-weight: 700;
+   
+}
+.nav li:hover {
+    color: blue;
+}
+
+/* Create three columns to store each element of main-page */
+.intro {
+    display: grid;
+    grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+    text-align: center ;
+}
+
+.intro h1 {
+    grid-row: 3/4;
+    grid-column: 4/5;  
+}
+.intro sub {
+    grid-row: 4/5;
+    grid-column: auto/span 7;
+    display: inline-block;
+    align-items: center;
+    font-size: 1.5em;
+    font-weight: bold;
+}
+
+.intro a {
+    grid-row: 5/6;
+    grid-column: 4/5;
+    font-size: 1.25em;
+    text-decoration: none;
+    color: black;
+    align-self: center;
+    border: solid lightgrey;
+    border-radius: 10px;
+    padding: 10px;
+    
+}
+
+.intro a:hover {
+    color: blue;
+}
+
+hr {
+    border-top: 1px solid #8c8b8b;
+    margin-top: 3%;
+    margin-bottom: 3%;
+}
+blockquote {
+    font-size: 1.2em;
+    font-style: italic;
+    font-family: 'Lucida Sans', sans-serif;
+    color: #555555;
+    width:60%;
+    margin: 20px auto;
+    padding:1.2em 30px 1.2em 75px;
+    line-height:1.6;
+    position: relative;
+   
+}
+  
+
+blockquote::before {
+    content: open-quote;
+}
+blockquote::after {
+    content: close-quote;
+}
+
+blockquote + p {
+    width:20%;
+    margin-left: 70%;
+    font-size: 1.2em;
+    font-family: 'Lucida Sans', sans-serif;
+    font-style: italic;
+}
+
+blockquote + p::before {
+    content: "--";
+}
+
+blockquote + p::after {
+    content: "--";
+}
+
+.why h2{
+    text-align: center;
+    margin-bottom: 3%;
+}
+
+.why ul {
+    display: flex; 
+    flex-flow: row wrap; 
+    justify-content: space-evenly;
+}
+
+.why li {
+    list-style-type: none;
+    padding: 0;
+    max-width: 25rem;
+    font-size: 1rem;
+}
+
+.why ul li img {
+    float: left;
+    padding: 3%;
+}
+
+/* create a team grid with 9 columns and 3 rows  */
+.team {
+    display:grid;
```

Comment:
I think a Flexbox could be a simpler implementation. Flexbox does lack the exactness from the wireframe but sort of accomplishes the same task in my opinion.

However, I think you this was great practice for you in implementing a grid and I think you did really well at the task. Great work!